test/spec/app_spec.rb

```diff
@@ -1066,21 +1066,98 @@
+      expect(@driver.page_source.include?('Meedan on Facebook')).to be(true)
 
-    # it "should find medias when searching by status" do
-    #   skip("Needs to be implemented")
-    # end
+      el = wait_for_selector("search-input", :id)
+      el.click
+      el.send_keys "video"
+      @driver.action.send_keys(:enter).perform
+      sleep 3 #due the reload
+      wait_for_selector("search-input", :id)
+      current = wait_for_selector_list("medias__item", :class).length
+      expect(old > current).to be(true)
+      expect(current > 0).to be(true)
+      expect(@driver.page_source.include?('weekly @Twitter video recap')).to be(true)
+      expect(@driver.page_source.include?('Meedan on Facebook')).to be(false)
 
-    # it "should find medias when searching by tag" do
-    #   skip("Needs to be implemented")
-    # end
+      el = wait_for_selector("search-input", :id)
+      el.clear
+      el.click
+      el.send_keys "meedan"
+      @driver.action.send_keys(:enter).perform
+      sleep 3 #due the reload
+      wait_for_selector("search-input", :id)
+      current = wait_for_selector_list("medias__item", :class).length
+      expect(old > current).to be(true)
+      expect(current > 0).to be(true)
+      expect(@driver.page_source.include?('Meedan on Facebook')).to be(true)
+      expect(@driver.page_source.include?('weekly @Twitter video recap')).to be(false)
+    end
 
-    # it "should move media to another project" do
-    #   skip("Needs to be implemented")
-    # end
+    it "should find medias when searching by status", bin2: true do
+      api_create_media_and_go_to_search_page
+      old = wait_for_selector_list("medias__item", :class).length
+      el = wait_for_selector("//div[contains(text(), 'False')]",:xpath)
+      el.click
+      sleep 3 #due the reload
+      wait_for_selector("search-input", :id)
+      current = wait_for_selector_list("medias__item", :class).length
+      expect(old > current).to be(true)
+      expect(current == 0).to be(true)
+      
+      old = wait_for_selector_list("medias__item", :class).length
+      el = wait_for_selector("//div[contains(text(), 'Unstarted')]",:xpath)
+      el.click
+      sleep 3 #due the reload
+      wait_for_selector("search-input", :id)
+      current = wait_for_selector_list("medias__item", :class).length
+      expect(old < current).to be(true)
+      expect(current == 1).to be(true)
+    end
+
+    it "should move media to another project", bin2: true do
+      data = api_create_team_and_project  
+      prj2 = api_create_project(data[:team].dbid.to_s)
+      p1 =  data[:project].team["projects"]["edges"][0]["node"]["title"]
+      p2 = prj2.team["projects"]["edges"][1]["node"]["title"]
+      source = api_create_media(data: data, url: "https://www.facebook.com/permalink.php?story_fbid=10155901893214439&id=54421674438")
+      @driver.navigate.to source.full_url
+      url1 = source.full_url
```

Comment:
`source` here should be media, no?

Reply:
Yes... It is just the var name. I'll change it.